orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/general/EqualTo.feature

```diff
@@ -5,351 +5,41 @@ Feature: User can specify that a value is equalTo a required value
 
 ### alone ###
 
-  Scenario: Running an 'equalTo' request that includes strings with roman numeric chars (0-9) only should be successful
+  Scenario: Running an 'equalTo' of a string should return only the string
```

Comment:
I guess a compromise between this and the large number of tests for different types of string could be an outline with a few types. Unsure how much value it would add though.